addons/aws/0.1.0/kubeadm-cluster-config-v1beta2.yml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: kubeadm.k8s.io/v1beta2
+kind: ClusterConfiguration
+metadata:
+  name: kubeadm-cluster-configuration
+clusterName: $KUBERNETES_CLUSTER_NAME
```

Comment:
remove this

Reply:
ok moved this config defaults in go as well.